src/main/java/uk/gov/hmcts/reform/pip/data/management/service/LocationService.java

```diff
@@ -241,12 +241,13 @@ public LocationDeletion deleteLocation(Integer locationId, String provenanceUser
                     requesterName = node.get("displayName").asText();
                 }
             } catch (JsonProcessingException e) {
+                log.error(e.getMessage());
                 throw e;
             }
             locationDeletion = checkActiveArtefactForLocation(location.get(), requesterName);
-            if (!locationDeletion.getIsExists()) {
+            if (Boolean.FALSE.equals(locationDeletion.getIsExists())) {
```

Comment:
Might be better to change this back to before and use primitive boolean for 'isExists' within LocationDeletion class. The constructor for that class is a primitive boolean but then the uppercase Boolean object is used for the class member.

Reply:
Agreed Kian, we discussed this before as well. It's fixed.